.github/workflows/integration-tests.yml

```diff
@@ -111,6 +111,7 @@ jobs:
           echo "\`${GITHUB_SHA}\`" >>$GITHUB_STEP_SUMMARY
 
   build-test-image:
+    if: startsWith(github.ref, 'refs/tags/') || github.event_name == 'schedule' || (github.event_name == 'push' && startsWith(github.ref, 'refs/tags/'))
```

Comment:
Are we able to trigger this on certain labels as well? It's a common workflow to open a PR when making config changes to long-running tests, and pull the image from that build.

Reply:
What label would you like to use?

Reply:
I added the ability to trigger this with the label 'build-test-image'